lib/Test2/Harness/Runner.pm

```diff
@@ -124,19 +147,16 @@ sub header {
     );
 
     open(my $fh, '<', $file) or die "Could not open file $file: $!";
-    my $ln = 0;
-    while (my $line = <$fh>) {
-        $ln++;
+
+    my $shbang = $self->_parse_shbang(<$fh>);
```

Comment:
This is something I thought about, and intended to test for, but never got around to it.